docs/openapi.yaml

```diff
@@ -131,7 +138,17 @@ paths:
             application/json:
               examples:
                 error_104_example:
-                  $ref: "#/components/examples/error_104_example"
+                  value:
+                    error_code: 104
+                    message: Invalid syntax
+                    details:
+                      [
+                        "Missing required fields",
+                        "Invalid email field format",
+                        "Invalid full_name field format",
```

Comment:
Format of password and full_name is what??

Reply:
e có để regex á a